packages/@vuepress/markdown/src/plugins/assetsPlugin.ts

```diff
@@ -1,4 +1,5 @@
 import type { PluginWithOptions } from 'markdown-it'
+import { decode } from 'mdurl'
```

Comment:
`mdurl` is not listed in the `dependencies` of markdown package.

We should add it explicitly.

Reply:
Got it, I forgot adding because it's already on the markdown-it toolchain.